src/components/ActionsMenu/dataHooks.ts

```diff
@@ -0,0 +1,3 @@
+export enum ACTIONS_MENU__DATA_KEYS {
```

Comment:
A bit picky, but we don't use `__` in our other `dataHooks` files.
So you can use just one `_`, for consistency.